### PR TITLE
document ticket endpoints

### PR DESCRIPTION
**Pull request by joaoinacio**

### CODE REVIEW

**Review by Jdww — DISMISSED** [review]

File: apiary.apib [hunk 1]
```diff
@@ -8105,6 +8108,263 @@ Update the current timer. Only possible if there is a timer running.
...
+## Tickets [/tickets]
+
+*Required scopes: `tickets`*
+
+### tickets.list [POST /tickets.list]
+
+Get a list of tickets.
+
++ Request (application/json)
+
```
Comment (tiagobastos) [inline apiary.apib]: Extra line
```suggestion
```

File: apiary.apib [hunk 2]
```diff
@@ -8105,6 +8108,263 @@ Update the current timer. Only possible if there is a timer running.
...
+        + id: `f29abf48-337d-44b4-aad4-585f5277a456` (string)
+        + subject (string)
+        + status (object)
+            + type: `ticketStatus` (string)
+            + id: `aba0ad66-bf59-49fa-b546-45dcbc5e7e6e` (string)
+        + assignee (object, nullable) - Null if unassigned
+            + type: `user` (string)
+            + id: `f29abf48-337d-44b4-aad4-585f5277a456` (string)
+        + created_at: `2017-05-09T11:25:11+00:00` (string)
+        + closed_at: `2017-05-09T11:25:11+00:00` (string, nullable)
```
Comment (tiagobastos) [inline apiary.apib:8180]: Maybe `closed_at` could also be part of the `.list`.. Thoughts?

**Review by tiagobastos — APPROVED** [review]

**Review by BrechtBonte — APPROVED** [review]

**Comment by MennoTammens** [comment]
In the web interface it's possible to connect a ticket to a project and milestone:
Screenshot 2024-01-29 at 13-12-12 Teamleader Focus

Will this also be added to the tickets endpoint? 

Use case:
`timeTracking.info` has a ticket as subject type, but it's impossible to find the project/milestone this timetracking is for.

**Comment by joaoinacio** [comment]
> Will this also be added to the tickets endpoint?
> 
> Use case: `timeTracking.info` has a ticket as subject type, but it's impossible to find the project/milestone this timetracking is for.

Thank you for your feedback. We are not providing that possibility at this stage, but may consider it for the future, keeping in mind that we are moving towards "New projects" (see https://developer.teamleader.eu/#/reference/new-projects).
As for the `timetracking.info` endpoint, we will also consider providing additional information about related entities other than the subject (related customer, project, etc).

Do feel free to reach out and send an email to [support@teamleader.eu](mailto:support@teamleader.eu) so that an internal ticket can be created. That way we can update you once this has been added to our API.

**Review by adognameddog — APPROVED** [review]

**Review by gdegirmenci — APPROVED** [review]

**Pull request merged**